src/lang/std/engineConnection.ts

```diff
@@ -1548,6 +1554,12 @@ export class EngineCommandManager {
       this.outSequence++
       this.engineConnection?.unreliableSend(command)
       return Promise.resolve()
+    } else if (cmd.type === 'export') {
+      const promise = new Promise((resolve, reject) => {
+        this.pendingExport = { resolve, reject }
+      })
```

Comment:
```typescript
 const promise = new Promise((resolve, reject) => {
    this.pendingExport = { resolve, reject, promiseTimeoutId: setTimeout(() => resolve(), 60_000) as any as number }
  })
```
The as any as number thing is annoying, there must be a way to tell typescript this is a browser not node 🤷‍♂️